clinica/pipelines/dwi_preprocessing_using_fmap/dwi_preprocessing_using_phasediff_fmap_workflows.py

```diff
@@ -104,3 +104,145 @@ def prepare_phasediff_fmap(name="prepare_phasediff_fmap"):
     # fmt: on
 
     return wf
+
+
+def compute_reference_b0(
+    low_bval: float,
+    use_cuda: bool,
+    initrand: bool,
+    output_dir=None,
+    name="compute_reference_b0",
+):
```

Comment:
Could you precise what the typing for what the pipeline returns ?

Reply:
Should have been addressed by [48ba105](https://github.com/aramis-lab/clinica/pull/893/commits/48ba105ce3f9b7d71fb4f8965bb8a1b0ca2e8c3a)
We'll need to check that the return value and the Optional type aren't breaking stuff when used through Nipype...